ksql-functional-tests/src/main/java/io/confluent/ksql/test/utils/Utilities.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.utils;
+
+import io.confluent.ksql.metastore.MetaStore;
+import io.confluent.ksql.parser.tree.HoppingWindowExpression;
+import io.confluent.ksql.parser.tree.KsqlWindowExpression;
+import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.SessionWindowExpression;
+import io.confluent.ksql.parser.tree.TumblingWindowExpression;
+import io.confluent.ksql.serde.Format;
+import io.confluent.ksql.serde.KsqlSerdeFactory;
+import io.confluent.ksql.test.serde.SerdeSupplier;
+import io.confluent.ksql.test.serde.avro.ValueSpecAvroSerdeSupplier;
+import io.confluent.ksql.test.serde.json.ValueSpecJsonSerdeSupplier;
+import io.confluent.ksql.test.serde.string.StringSerdeSupplier;
+import io.confluent.ksql.test.tools.exceptions.InvalidFieldException;
+import io.confluent.ksql.util.KsqlException;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public final class Utilities {
```

Comment:
nit: How about splitting this class into `SerdeUtil` and `WindowUtil` (and separating the methods appropriately) to feel more organized? :)